gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Jan 08 14:15:45 CET 2020
+#Sun Sep 12 16:42:40 CEST 2021
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
7.1.1

Reply:
Like I said above, how about I update this after migrating to Maven?